p4broker/hooks/init

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -xe
+#
+
+if [ ! -d {{pkg.svc_data_path}}/log ]; then
+	mkdir -p {{pkg.svc_data_path}}/log
+	chown -R hab:hab {{pkg.svc_data_path}}/log
```

Comment:
iirc you shouldn't need to chown here. However, if I'm wrong (totally possible :joy:), you'll want to use `{{pkg.svc_user}}:{pkg.svc_group}}`

Reply:
I've hit some weird issues with ownership. This was my lazy way of dealing with it.